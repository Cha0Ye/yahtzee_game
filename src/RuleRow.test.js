import React from 'react';
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import RuleRow from './RuleRow';

// smoke test
it('renders without crashing', () => {
  mount(<RuleRow />);
});

// snapshot test rulerow
it("matches snapshot for rulerow", function() {
	const doScoreFn = jest.fn();
	let wrapper = mount(<RuleRow name="TEST" score={2}
					doScore={doScoreFn}/>);
	let serialized = toJson(wrapper);
	expect(serialized).toMatchSnapshot();
});


