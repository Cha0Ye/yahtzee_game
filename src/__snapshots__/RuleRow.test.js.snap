// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for rulerow 1`] = `
<RuleRow
  doScore={[Function]}
  name="TEST"
  score={2}
>
  <tr
    className="RuleRow RuleRow-active"
    onClick={[Function]}
  >
    <td
      className="RuleRow-name"
    >
      TEST
    </td>
    <td
      className="RuleRow-score"
    >
      2
    </td>
  </tr>
</RuleRow>
`;
